SPECS/xha.spec

```diff
@@ -18,6 +18,9 @@ BuildRequires: xen-devel
 
 Requires: portreserve
 
+# XCP-ng patches
+Patch1000: xha-10.1.0-supports-drbd.XCP-ng.patch
```

Comment:
name: "support" would be the imperative form as commonly used in commit messages and patch names, rather than "supports"